.github/container/test-pax.sh

```diff
@@ -422,5 +451,31 @@ else
     $([[ $MULTIPROCESS != 0 ]] && echo --multiprocess_gpu)
 fi
 
+echo "Checking for FMHA instructions in HLO!"
+
+if [[ "$ENABLE_FMHA" -eq "1" ]]; then 
+    echo "Inside if Statement!"
+    ## Check if fmha instructions are present in the HLO dumped file or not.
+    fmha_regex="fmha[-bmm]?[-scale]?[-bias]?[-mask]?[-softmax]?[-dropout]?[-bmm]?[-backward]?*"
+    result=$(grep -irlnE "$fmha_regex" "${HLO_DIR}/"*.txt)
+
+    if [[ $SAVE_HLO -eq 0 ]]; then
```

Comment:
I would expect to skip saving if `$SAVE_HLO == 0`, rather than delete it afterwards.

Reply:
for fmha the hlo is dumped in output folder by default to search for fmha instructions. The idea is to delete the hlo folder if user doesn't want to have hlo folder inside the output folder which is saved as artifact.